k8s/openebs-operator.yaml

```diff
@@ -37,7 +37,13 @@ rules:
   verbs: [ "get", "list", "create" ]
 - apiGroups: ["*"]
   resources: ["storagepools"]
-  verbs: ["get", "list"] 
+  verbs: ["get", "list"]
+- apiGroups: ["volumesnapshot.external-storage.k8s.io"]
+  resources: ["volumesnapshots"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment:
We decided to use grpc for snapshots, right? Is it required to have 'watch'?

Reply:
The gRPC comes between the provisioner to API Server or volume controller. These permissions are to allow the openebs-snapshot-controller to receive requests from the kubectl command.

Reply:
ok